wgs/community-membership.md

```diff
@@ -8,10 +8,12 @@ Responsibilities for most roles are scoped to these subprojects.
 
 | Role | Responsibilities | Requirements | Defined by |
 | -----| ---------------- | ------------ | -------|
-| member | active contributor in the community | sponsored by 2 reviewers.  multiple contributions to the project. | Kubernetes GitHub org member. |
-| reviewer | review contributions from other members | history of review and authorship in a subproject | [OWNERS] file reviewer entry. |
-| approver | approve accepting contributions | highly experienced and active reviewer + contributor to a subproject | [OWNERS] file approver entry|
-| subproject owner | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgement for the subproject | [wgs.yaml] subproject [OWNERS] file *owners* entry |
+| Member | Active contributor in the community | Sponsored by 2 reviewers.  Multiple contributions to the project. | GitHub org member. |
+| Scribe | Ensure important information is represented in working group notes | Membership | Write access to WG documentation | 
+| Reviewer | Review contributions from other members | History of review and authorship in a subproject | [OWNERS] file reviewer entry. |
+| Approver | Approve accepting contributions | Hghly experienced and active reviewer | [OWNERS] file approver entry|
```

Comment:
```suggestion
| Approver | Approve accepting contributions | Hoghly experienced and active reviewer | [OWNERS] file approver entry |
```

Reply:
I think we both misspelled this one, I'll correct it next push. 😄